src/crud.js

```diff
@@ -33,32 +33,35 @@ models: {
 
 export default (server, model, { prefix, defaultConfig: config, models: permissions }) => {
   const modelName = model._singular;
+  const permissionsIsArray = Array.isArray(permissions);
```

Comment:
Push up a change to do this. I like it a lot because the logic chain now ends with an `else` instead of a trailing `else if`. 👍